modules/embit/module.cpp

```diff
@@ -12,9 +11,14 @@ namespace bitcoinfuzz
 
         std::optional<bool> Embit::miniscript_parse(std::string str) const
         {
-            bool result = embit_miniscript_miniscript_parse(str.c_str());
+            bool result = embit_miniscript_parse(str.c_str());
             return result;
         }
 
+        std::optional<bool> Embit::descriptor_parse(std::string str) const
+        {
+            bool result = embit_descriptor_parse(str.c_str());
```

Comment:
For both descriptor and miniscript parse you can return embit_descriptor_parse/embit_miniscript_parse directly. No need to have the `bool result`.